de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/umsatzreport_bpartner/report.jrxml

```diff
@@ -381,7 +380,9 @@
 	</columnHeader>
 	<detail>
 		<band height="12" splitType="Stretch">
-			<printWhenExpression><![CDATA[new Boolean ( $F{unionorder}.intValue() == 1 )]]></printWhenExpression>
+			<printWhenExpression><![CDATA[new Boolean ( $F{unionorder}.intValue() == 1 )
+&&  (BigDecimal.ZERO.compareTo($F{sameperiodsum}) !=0
```

Comment:
this seems needlessly complicated: why not `$F{sameperiodsum}.signum() != 0 || $F{compperiodsum}.signum() != 0`
 ?